charts/controller/README.md

```diff
@@ -30,6 +30,16 @@ The command deploys Rookout on the Kubernetes cluster in the default configurati
 
 > **Tip**: List all releases using `helm list`
 
+## Installation without helm
+If you're not using helm tiller on your kubernetes, you'll still be able to install the controller. Helm will be needed to be installed locally just to create the yaml file from the templates.
+
+1.  Install helm locally: https://helm.sh/docs/intro/install/ 
+2.  Clone this repository and `cd charts/controller`
+3.  run ``` helm template . --set controller.token=YOUR_ORGANIZATIONAL_TOKEN --name=rookout```
+4.  A generation of the yamls will be presented so copy them to a single file, called `rookout-controller.yaml`, for example 
```

Comment:
Instead of asking the user to copy, you can ask them to pipe the output to the file, e.g. `helm template . --set controller.token=YOUR_ORGANIZATIONAL_TOKEN --name=rookout > rookout-controller.yaml`